admin_manual/installation/server_tuning.rst

```diff
@@ -146,9 +164,30 @@ To disable the revalidation completely:
 
   opcache.validate_timestamps = 0
 
-Any change to ``config.php`` will then require either restarting PHP, manually clearing the cache, or invalidating this particular script.
+Any Server/app upgrades or changes to ``config.php`` will then require restarting PHP (or otherwise manually clearing the cache or invalidating this particular script).
```

Comment:
```suggestion
Any Server/app upgrades via ``occ`` command or changes to ``config.php`` will then require restarting PHP (or otherwise manually clearing the cache or invalidating this particular script).
```

Reply:
That's correct. If you feel it's important to be 100% correct, then please shoot another pull request. My feeling is that a good number of people turned off the validation but don't understand the implications, and therefore I think it's fine to suggest a restart in any case.